assets/dev/js/editor/controls/slider.js

```diff
@@ -200,8 +200,15 @@ ControlSliderItemView = ControlBaseUnitsItemView.extend( {
 			return;
 		}
 
+		const currentSize = this.getSize(),
+			alreadyConverted = 'string' === typeof currentSize && currentSize.includes( 'fr' );
```

Comment:
could this check replace `isFrToCustom` on 199 line ?

Reply:
@mykytamurzin Nope it can't. If I remove `isFrToCustom`, then when the user switches from `px -> custom` it will also run the logic. We don't know what the previous unit was at this point in execution, which is why we are checking that the control should only have `fr`, `custom` for this logic to run.